### DISCUSSION TITLE
Music sheet SVG displays broken

### DISCUSSION THREAD

**Discussion by KamilDev** [dlemstra/Magick.NET#1768] [Help]
### Magick.NET version

v14.2.0 Magick.NET-Q16-AnyCPU

### Environment (Operating system, version and so on)

Windows 11

### Description

The following music sheet SVG displays broken:
![music-sheet](https://github.com/user-attachments/assets/41d7f00b-c83c-4322-bb41-b2610c957dcd)

When displayed using Magick.NET:
![image](https://github.com/user-attachments/assets/822a9216-6625-4e4b-b7c6-6fb863d81c82)

When displayed using a different library such as Svg.Skia or SharpVectors, it's displayed correctly.

### Steps to Reproduce

1. [Download the music-sheet.svg](https://github.com/user-attachments/assets/41d7f00b-c83c-4322-bb41-b2610c957dcd)
2. Display it using Magick. For example:
```cs
 public void DisplaySvg(string svgPath)
 {
     using (var image = new MagickImage(svgPath))
     {
         // Convert to PNG for display
         var memoryStream = new MemoryStream();
         image.Write(memoryStream, MagickFormat.Png);
         memoryStream.Position = 0;

         var bitmap = new BitmapImage();
         bitmap.BeginInit();
         bitmap.CacheOption = BitmapCacheOption.OnLoad;
         bitmap.StreamSource = memoryStream;
         bitmap.EndInit();
         bitmap.Freeze();

         SvgImage.Source = bitmap;
         DropHint.Visibility = Visibility.Collapsed;
     }
 }
```

[Full project here.](https://github.com/KamilDev/WpfSvgDisplay)

**Comment by dlemstra** [comment]
The internal SVG coder sometimes doesn't understand some features in svg files. I would advise you to install Inkscape to get better results.

p.s. I have a created a library called `Magick.NET.SystemWindowsMedia ` that can be used to create a `BitmapSource`.

**Reply by KamilDev** [reply]
Magick.NET is being utilized in [QuickLook](https://github.com/QL-Win/QuickLook) to preview SVG files, I assume Inkscape is not applicable here.

Is there any chance of this being fixed, should I raise the issue in ImageMagick repo?
